src/scitokens/scitokens.py

```diff
@@ -470,8 +471,8 @@ def _validate_jti(self, value):
         JTI, or json token id, should always pass.  It's mostly used for logging
         and auditing.
         """
-        # Fix for unused argument
-        del value
+        logger = logging.getLogger('scitokens')
```

Comment:
can we make this at the "module" level instead?  I don't know how expensive this call is.